src/components/tier.jsx

```diff
@@ -63,12 +44,21 @@ class Tier extends React.Component {
 
 
   render() {
+
+    const items = this.state.items;
+
+    let itemsArray = [];
+
+    for (let key in items) {
+      let itemId = items[key];
+      itemsArray.push(<Item key={itemId} itemId={itemId}/>)
+    }
+
     return (
       <div>
-        <div className="item">
+        <div className="tier ${this.state.tierId">
           Tier {this.state.tierId}
-          <Item itemId={this.state.items.item1}/>
-          <Item itemId={this.state.items.item2}/>
+          {itemsArray}
```

Comment:
Is this rendering? The way I am reading it, you are handing this component an array to be rendered as an array. I could be mistaken. But I think that array needs to be mapped somehow.

Reply:
It's handled automagically because each item in the array is a component. I could use map instead of a for loop to generate the array, which would be cleaner (and I will probably end up doing now you brought it up). But the array itself will be pulled apart into a list of the components inside of it.